packages/calypso-products/src/plans-list.tsx

```diff
@@ -2859,6 +2865,27 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		],
 	},
 
+	[ PLAN_JETPACK_SECURITY_T1_BI_YEARLY ]: {
+		...getPlanJetpackSecurityT1Details(),
+		...getBiAnnualTimeframe(),
+		getStoreSlug: () => PLAN_JETPACK_SECURITY_T1_BI_YEARLY,
+		getPathSlug: () => 'security-20gb-yearly',
```

Comment:
It seems like we have different paths for the yearly and monthly plans, do we need a new one for this new billing term?
I'm not sure where that is being used tho

Reply:
Thanks for reviewing Ian!!   Yeah, it's strange, it looks like `getPathSlug()` is only being used in a couple of places, and only for wpcom products, so for Jetpack products this value is not used or associated with anything.  I'll change it to match format of all other Jetpack products. 👍